docs/guides/1.0.0-migration-guide.md

```diff
@@ -0,0 +1,58 @@
+---
+page_title: "Fastly Terraform 1.0.0 Migration Guide"
+---
+
+## Fastly Terraform 1.0.0 Migration Guide
+
+**Resources omit `_v1` suffix**:
+
+All resources now have a consistent naming convention which omit the previous v1 suffix.
+
+- `fastly_service_v1` -> `fastly_service_vcl` (renamed with `_vcl` suffix)
+- `fastly_service_acl_entries_v1` -> `fastly_service_acl_entries`
+- `fastly_service_dictionary_items_v1` -> `fastly_service_dictionary_items`
+- `fastly_service_dynamic_snippet_content_v1` -> `fastly_service_dynamic_snippet_content`
+- `fastly_user_v1` -> `fastly_user`
+
```

Comment:
**Can you add the following (before the "Logging resources have consistent naming format:" section). This was a later addition to the [release notes migration guide](https://github.com/fastly/terraform-provider-fastly/releases/tag/v1.0.0)...**

When switching to the new 1.0.0 release, some existing projects might see an error such as:

```
Error: no schema available for module.fastly.<...> while reading state; this is a bug in Terraform and should be reported
```

This error is because the Terraform state file will have a different resource name to what's now exposed via the 1.0.0 release. To resolve this issue you'll need to first remove the old resource from the state file and then reimport the data using the new resource name.

As an example, if you had defined an `fastly_service_acl_entries_v1` resource, then you could remove and reimport using the following commands (any brackets `<...>` should be replaced with actual values):

```bash
terraform state rm 'fastly_service_acl_entries_v1.<your_resource_name>["<your_acl_name>"]'
terraform import 'fastly_service_acl_entries.<your_resource_name>["<your_acl_name>"]' <service_id>/<acl_id>

terraform state rm 'fastly_service_v1.<your_resource_name>'
terraform import 'fastly_service_vcl.<your_resource_name>' <service_id>
```

It's also recommended that you keep a backup of your state file so you can use it to reference any resource IDs necessary.